tests/app/delivery/test_send_to_providers.py

```diff
@@ -10,7 +10,8 @@
 
 
 import app
-from app import aws_sns_client, mmg_client
+from app import aws_sns_client, mmg_client, clients
```

Comment:
I think `aws_sns_client` needs to be removed here to get the failing tests to pass, and it should be removed from some of the other tests in this file. Since we're no longer loading it into app, it can't be mocked like in `test_should_not_send_to_provider_when_status_is_not_created`